node/src/components/storage.rs

```diff
@@ -173,6 +174,92 @@ pub enum Error {
     /// Error when validating a block.
     #[error(transparent)]
```

Comment:
The doc comment for this `Error` enum should be changed (I can't attach this comment to the actual relevant line: it's line 135)

Reply:
Fixed in 897417c96